index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Website</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+<body>
+  <main>
+    <header>
+      <nav>
+        <div class="logo">My Logo</div>
+        <div id="menu-btn">
+          <div class="menu-bar"></div>
+          <div class="menu-bar"></div>
+          <div class="menu-bar"></div>
+        </div>
```

Comment:
- [ ] Good job so far 👍🏽 . Please note that only the hamburger ought to be wrapped inside a `nav` tag and not the logo because the hamburger will contain the navigation elements so it s semantic to use the `nav`  tag for that. Please remove your logo from inside the` nav` tag and just let it be in the `header` tag 👍🏽